webapp/jaasai/views.py

```diff
@@ -95,7 +104,8 @@ def support():
 @jaasai.route("/blog/feed")
 def blog_feed():
     feed_url = "https://admin.insights.ubuntu.com/tag/juju/feed"
-    feed = feedparser.parse(feed_url)
+    response = cached_session.get(feed_url, timeout=3)
```

Comment:
what is 3? Can you add a comment.

Reply:
Added

Reply:
`cached_session` should have a default timeout setting, I think. Did you consider using the default?

Reply:
I think requests doesn't set a timeout by default (http://docs.python-requests.org/en/master/user/advanced/#timeouts).

Reply:
Oh! You're using `requests_cache` directly. You should be using `canonicalwebteam.http`, which sets a default timeout:

https://github.com/canonical-web-and-design/http/blob/master/canonicalwebteam/http/__init__.py#L62

So that's half a second to start receiving data, and 3 seconds for the whole request to complete.

I see this is merged now, but do you think you could start using this module instead, and use the default timeout rather than overriding it? Unless there's a good reason not to.

Reply:
I've created an issue so this doesn't get lost. https://github.com/canonical-web-and-design/jaas.ai/issues/180

Reply:
Thanks @hatched